### PR TITLE
refactoring/1171 naming cleanup

### PR DESCRIPTION
**Pull request by RandomTannenbaum**

### CODE REVIEW

File: backend/src/main/java/ch/puzzle/okr/dto/overview/DashboardDto.java [hunk 1]
```diff
@@ -2,5 +2,5 @@
 
 import java.util.List;
 
-public record DashboardDto(Boolean adminAccess, List<OverviewDto> overviews) {
+public record DashboardDto(Boolean isAdminAccess, List<OverviewDto> overviews) {
```
Comment (RandomTannenbaum) [inline backend/src/main/java/ch/puzzle/okr/dto/overview/DashboardDto.java:5]: Check if still needed

File: backend/src/test/java/ch/puzzle/okr/service/persistence/CheckInPersistenceServiceIT.java [hunk 2]
```diff
@@ -66,13 +66,12 @@ void getLastCheckInOfKeyResultShouldGetLastCheckInOfKeyResult() {
...
         var allCheckins = checkInPersistenceService.getCheckInsByKeyResultIdOrderByCheckInDateDesc(KEY_RESULT_ID);
         assertLastIsCreatedAfterAllOtherCheckIns(lastCheckIn, allCheckins);
-
     }
 
     private void assertLastIsCreatedAfterAllOtherCheckIns(CheckIn last, List<CheckIn> allCheckIns) {
-        for (CheckIn checkInLoop : allCheckIns) {
-            if (!Objects.equals(checkInLoop.getId(), last.getId())) {
-                assertTrue(last.getCreatedOn().isAfter(checkInLoop.getCreatedOn()));
+        for (CheckIn checkIn : allCheckIns) {
```
Comment (ManuelMoeri) [inline backend/src/test/java/ch/puzzle/okr/service/persistence/CheckInPersistenceServiceIT.java:72]: Nice!

File: backend/src/test/java/ch/puzzle/okr/util/quarter/check/QuarterRangeCheckerTest.java [hunk 3]
```diff
@@ -40,10 +40,10 @@ void nowIsInQuarterShouldReturnTrueIfNowMatchesQuarterStartDate() {
         LocalDate startDate = LocalDate.of(2024, 7, 1);
         Quarter quarter = quarter(startDate, null);
 
-        LocalDate nowMatchesQuarterStartDate = LocalDate.of(2024, 7, 1);
+        LocalDate nowMatchingQuarterStartDate = LocalDate.of(2024, 7, 1);
```
Comment (ManuelMoeri) [inline backend/src/test/java/ch/puzzle/okr/util/quarter/check/QuarterRangeCheckerTest.java:43]: Why were all these attributes renamed? Is there something wrong with the old name?

**Review by ManuelMoeri — CHANGES_REQUESTED** [review]
I have only done the backend in this review. The frontend will need to be done separately. Ⓜ️Ⓜ️

File: backend/src/main/java/ch/puzzle/okr/controller/CheckInController.java [hunk 4]
```diff
@@ -15,7 +15,7 @@
 import org.springframework.web.bind.annotation.*;
 
 @RestController
-@RequestMapping("api/v2/checkIns")
+@RequestMapping("api/v2/checkins")
```
[the remark this answers is not in the document]
Comment (RandomTannenbaum) [reply backend/src/main/java/ch/puzzle/okr/controller/CheckInController.java:18]: Only in long names

File: backend/src/main/java/ch/puzzle/okr/service/business/ObjectiveBusinessService.java [hunk 5]
```diff
@@ -61,15 +61,15 @@ public Objective updateEntity(Long id, Objective objective, AuthorizationUser au
     }
 
     private void setQuarterIfIsImUsed(Objective objective, Objective savedObjective) {
-        boolean imUsed = isImUsed(objective, savedObjective);
-        if (imUsed) {
+        boolean isImUsed = isImUsed(objective, savedObjective);
```
[the remark this answers is not in the document]
Comment (RandomTannenbaum) [reply backend/src/main/java/ch/puzzle/okr/service/business/ObjectiveBusinessService.java:72]: IMUsed is http response code 226. Also the method we call is also called `isImUsed`.

File: backend/src/test/java/ch/puzzle/okr/util/quarter/check/QuarterRangeCheckerTest.java (hunk 3 above)
Comment (RandomTannenbaum) [reply backend/src/test/java/ch/puzzle/okr/util/quarter/check/QuarterRangeCheckerTest.java:43]: Matches sounds like a boolean to me, but ig thats subjective

File: frontend/src/app/components/action-plan/action-plan.component.html [hunk 6]
```diff
@@ -1,23 +1,17 @@
...
     <div class="action-point-item" *ngFor="let action of (control | async)!; index as i; trackBy: trackByFn" cdkDrag>
       <div class="drag-drop-dots">
-        <img
-          cdkDragHandle
-          class="icons"
-          src="../../../assets/icons/drag_indicator.svg"
-          alt="Drag indicator"
-          [attr.data-testId]="'dragIcon'"
-        />
+        <img cdkDragHandle class="icons" src="../../../assets/icons/drag_indicator.svg" alt="Drag indicator" />
```
Comment (peggimann) [inline frontend/src/app/components/action-plan/action-plan.component.html:14]: why is the test-id removed?

File: frontend/src/app/components/application-top-bar/application-top-bar.component.html [hunk 7]
```diff
@@ -30,10 +30,10 @@
         (menuOpened)="menuIsOpen = true"
         [attr.data-testId]="'user-options'"
         [matMenuTriggerFor]="menu"
-        class="topBarEntry flex-nowrap btn"
+        class="top-bar-entry flex-nowrap btn"
       >
-        <mat-icon aria-hidden="false" aria-label="person icon" fontIcon="person_outline" id="topbar-user"></mat-icon>
-        <span [attr.data-testId]="'user-name'" class="fw-normal fs-6 d-none d-md-flex">{{ userFullName }}</span>
+        <mat-icon aria-hidden="false" aria-label="person icon" fontIcon="person_outline" id="top-bar-user"></mat-icon>
+        <span [attr.data-testId]="'username'" class="fw-normal fs-6 d-none d-md-flex">{{ userFullName }}</span>
```
Comment (peggimann) [inline frontend/src/app/components/application-top-bar/application-top-bar.component.html:36]: to remain consistend user-name should be in kebab case

File: frontend/src/app/shared/custom/scoring/scoring.component.ts [hunk 8]
```diff
@@ -33,16 +33,13 @@ export class ScoringComponent implements OnInit, AfterViewInit, OnChanges {
   targetPercent: number = 0;
   labelPercentage: Observable<number>;
   stretched: boolean = false;
-  protected readonly isLastCheckInNegative = isLastCheckInNegative;
```
Comment (peggimann) [inline frontend/src/app/shared/custom/scoring/scoring.component.ts:36]: why is this removed? don't we need this for the exclamation mark to be set for negative checkins?

File: frontend/src/app/shared/types/menu-entry.ts [hunk 9]
```diff
@@ -1,11 +0,0 @@
-import { ComponentType } from '@angular/cdk/overlay';
```
Comment (peggimann) [inline frontend/src/app/shared/types/menu-entry.ts:1]: are we sure this is no longer needed?

**Review by peggimann — CHANGES_REQUESTED** [review]

File: frontend/src/app/shared/types/menu-entry.ts (hunk 9 above)
Comment (kcinay055679) [reply frontend/src/app/shared/types/menu-entry.ts:1]: yes

File: frontend/src/app/shared/custom/scoring/scoring.component.ts (hunk 8 above)
Comment (kcinay055679) [reply frontend/src/app/shared/custom/scoring/scoring.component.ts:36]: looks like this got renamed to calculateCurrentPercentage with a check for <1

File: frontend/src/app/components/action-plan/action-plan.component.html (hunk 6 above)
Comment (kcinay055679) [reply frontend/src/app/components/action-plan/action-plan.component.html:14]: i guess it got removed because it had 0 ussages

File: frontend/src/app/components/application-top-bar/application-top-bar.component.html (hunk 7 above)
Comment (kcinay055679) [reply frontend/src/app/components/application-top-bar/application-top-bar.component.html:36]: It got defined in [here](https://codimd.puzzle.ch/DWVLjwB8SkaCw0-0LbREWg) as one word

**Comment by kcinay055679** [comment]
got closed in favor of the rebased version #1263
